src/layout/layout.d.ts

```diff
@@ -52,6 +52,7 @@ export interface IComponentRadioOrCheckbox<T extends Extract<ComponentTypes, 'Ra
   extends ILayoutCompBase<T>,
     ISelectionComponent {
   layout?: LayoutStyle;
+  showAsCard?: boolean;
```

Comment:
Will the Checkboxes-component now also get this property? If so i think we should not add it to the Checkboxes-component until this is implemented as a feature.

Reply:
Ah yes. Will have a look and see if I can fix this issue.